features/publishable/publishable.feature

```diff
@@ -0,0 +1,97 @@
+Feature: Access to unpublished/draft resources should be configurable
+  In order to restrict access to draft components
+  As a an application developer
+  I must be able to configure the ability to access the resource
+
+  Background:
+    Given I add "Accept" header equal to "application/ld+json"
+    And I add "Content-Type" header equal to "application/ld+json"
+
+  # GET collection
+  @createSchema
+  Scenario: As a user with draft access, when I get a collection of published resources with draft resources available, it should include the draft resources instead of the published ones.
+
+  @createSchema
+  Scenario: As a user with draft access, when I get a collection of published resources with draft resources available, and published=true query filter, it should include the draft resources instead of the published ones.
+
+  @createSchema
+  Scenario: As a user with no draft access, when I get a collection of published resources with draft resources available, it should not include the draft resources.
```

Comment:
maybe? - it should include the published resources instead of the draft ones, and no draft resources should there be no published version available